### PR TITLE
Add CF labels

### PR DESCRIPTION
**Pull request by heycait**
[#174735575]

Do these make sense?? This is from the guide here: https://github.com/cloudfoundry-incubator/kubernetes-guidelines

This replaced the `app: metric-proxy` label so we may need to update cf-for-k8s to replace it with the new label. Or for now, I could add it back in?

### CODE REVIEW

**Comment by heycait** [comment]
Looks like I need to fix how the method works. Not getting picked up correctly in ytt.

File: hack/bump-cf-for-k8s.sh [hunk 1]
```diff
@@ -10,3 +10,7 @@ pushd "${CF_FOR_K8s_DIR}"
   vendir sync -d config/metrics/_ytt_lib/metric-proxy="${REPO_DIR}/config"
 popd
 
+# sed -i "s|version:.*|version: dev-test|" $REPO_DIR/config/values.yml
+
+# On Mac
```
Comment (kkburr) [inline hack/bump-cf-for-k8s.sh:15]: Can you give a short description of what this is doing? Also, did you mean to leave the comment above?

Comment (heycait) [reply hack/bump-cf-for-k8s.sh:15]: As in add it to the code as well?

This is modifying the `config/values.yml` to ensure that the version number is filled out during testing. I didn't mean to commit `version: dev-test` within the file itself. It actually should be blank.

I was seeing errors when this wasn't filled and resulted in some type of error. I can't remember what it was or maybe I was conflating errors.

Comment (kkburr) [reply hack/bump-cf-for-k8s.sh]: What I meant was adding "This is modifying the config/values.yml to ensure that the version number is filled out during testing." as a comment in the code in addition to #On Mac.

I think you're right that we do need it when running locally, although, maybe we can just leave it blank for now and add it to the script as a separate commit.

**Comment by kkburr** [comment]
LGTM 🚢  :shipit:

File: hack/bump-cf-for-k8s.sh (hunk 1 above)
Comment (kkburr) [reply hack/bump-cf-for-k8s.sh]: Just saw your last comment 👍

**Pull request merged**